byline-manager-sst.php

```diff
@@ -11,4 +11,129 @@
  * @package         Byline_Manager_SST
  */
 
-// Your code starts here.
+namespace Byline_Manager\SST;
+
+use WP_Error;
+use WP_Post;
+
+if ( ! method_exists( '\Byline_Manager\Utils', 'set_post_byline' ) ) {
+	return;
+}
+
+add_action( 'rest_api_init', __NAMESPACE__ . '\add_rest_field' );
+
+/**
+ * Register the byline_entries REST field to the sst-post object.
+ */
+function add_rest_field() {
+	register_rest_field(
+		'sst-post',
+		'byline_entries',
+		[
+			'update_callback' => function( array $byline_entries, WP_Post $post ) {
+				$meta = [
+					'byline_entries' => [],
+				];
+
+				foreach ( $byline_entries as $entry ) {
+					// Don't save empty items.
+					if ( empty( $entry['text'] ) && empty( $entry['name'] ) ) {
```

Comment:
Should this be an or comparison based on the error message?

Reply:
Nope, one is required. If they're both empty, there's nothing to work with